views/imagestream-listing.html

```diff
@@ -1,4 +1,4 @@
-<table class="listing-ct">
+<table ng-class="classNames ? '{{classNames}}' : 'listing-ct'">
```

Comment:
I have.  The [screenshot above](https://github.com/openshift/registry-image-widgets/pull/18#issue-247518924) utilizes the change in the context of origin-web-console.  In the case of the console, we appear to not be utilizing all of the functionality of the directive, so it's not a problem.  My ultimate aim is to simply fix https://github.com/openshift/origin-web-console/issues/1849, which this change enables.